pandas/tests/io/json/test_normalize.py

```diff
@@ -85,7 +85,52 @@ def missing_metadata():
     ]
 
 
-class TestJSONNormalize:
+@pytest.fixture
+def deep_nested_inconsistantly_formatted():
+    return [{
+            'keyA': 1.11,
+            'keyB': False,
+            'keyC': [{
+                     'keyCA': 'StringCA1',
+                     'keyCB': {
+                               'keyCBA': 4,
+                               'keyCBB': 5,
+                               'keyCBC': [{'keyCBCA': 6, 'keyCBCB': 7, 'keyCBCC': 8.2},
+                                          {'keyCBCA':'keyCBCA', 'keyCBCB': 10, 'keyCBCC': 11},
+                                          {'keyCBCA': 12, 'keyCBCB': [13], 'keyCBCC': 14}],
+                               'keyCBD': 15
+                               },
+                     'keyCC': 16
+                     }],
+            'keyD': 17,
+            'keyE': [{
+                     'keyEA': 18,
+                     'keyEB': {'keyEBA': 19,'keyEBB': 20}
+                     }]
+            },{
+            'keyA': 31.2,
+            'keyB': True,
+            'keyC': [{
+                    'keyCA': {'StringCA2': 'StringCA2'},
+                    'keyCB': {
+                              'keyCBA': 34,
+                              'keyCBB': 35,
+                              'keyCBC': [{'keyCBCA': 'keyCBCA', 'keyCBCB': 37.1, 'keyCBCC': 38},
+                                         {'keyCBCA': 39, 'keyCBCB': True, 'keyCBCC': 41},
+                                         {'keyCBCA': 42, 'keyCBCB': 43, 'keyCBCC': {'test':44}}],
+                              'keyCBD': 45
+                              },
+                    'keyCC': False
+                    }],
+            'keyD': 47,
+            'keyE': [{
+                    'keyEA': 48,
+                    'Missing keyEB': 49
+                    }]
+            }]
+
+
+class TestJSONNormalize(object):
```

Comment:
I think the 2.X new style class is going to fail CI; should just remove `(object)` altogether

Reply:
Corrected